.travis.yml

```diff
@@ -47,14 +47,13 @@ matrix:
         - pip3 install -r requirements.txt
         - brew link gcc
         - export PATH="/usr/local/bin:$PATH"
-        - ln -s /usr/local/bin/gcc-8 /usr/local/bin/gcc
```

Comment:
This results in you running Clang instead of GCC

Reply:
good catch, thanks.